src/toil/jobStores/awsJobStore.py

```diff
@@ -389,6 +408,31 @@ def _parseArgs(cls, jobStoreString):
 
         return region, namePrefix
 
+    def _toSDBItem(self,file, isProtected=True):
+        item = {}
+        bytes_per_attr = 1024 - encryptionOverhead
+        shouldEncrypt = bool(isProtected and self.sseKey)
+        encryptMethod = encrypt if shouldEncrypt else lambda string, path: string
```

Comment:
Make this a simple if statement instead of this conditional expression with lambda. That also takes care of the overhead being added unconditionally.